src/main/java/net/masterthought/cucumber/reducers/ReportFeatureMerger.java

```diff
@@ -0,0 +1,20 @@
+package net.masterthought.cucumber.reducers;
+
+import net.masterthought.cucumber.json.Feature;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+public interface ReportFeatureMerger extends Predicate<List<ReducingMethod>> {
+
+    /**
+     * @return list of features which are organized by merger.
+     *
+     * @see net.masterthought.cucumber.reducers.ReportFeatureAppendableMerger
+     * @see net.masterthought.cucumber.reducers.ReportFeatureByIdMerger
+     * @see net.masterthought.cucumber.reducers.ReportFeatureWithRetestMerger
+     *
+     * Merger's type depends on a ReducingMethod which is coming from the configuration.
+     */
+    List<Feature> merge(List<Feature> features);
+}
```

Comment:
worth to redefine https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html#test-T- here with documentation what do you expect from Feature point of view